app/controllers/concepts_controller.rb

```diff
@@ -24,7 +25,10 @@ def edit
   # POST /concepts
   # POST /concepts.json
   def create
+    # byebug
```

Comment:
Don't know the policy on keeping debug stuff in but commented out, do we have one?